platform/windows/src/headless_backend_osmesa.cpp

```diff
@@ -14,19 +14,23 @@ namespace gl {
 
 class OSMesaBackendImpl final : public HeadlessBackend::Impl {
 public:
-    OSMesaBackendImpl() {
-        context = OSMesaCreateContextAttribs(std::initializer_list<int>({OSMESA_FORMAT,
+    OSMesaBackendImpl()
+        : buffer(std::make_unique<uint8_t[]>(2048 * 2048 * 4)),
+          context(OSMesaCreateContextAttribs(std::initializer_list<int>({OSMESA_FORMAT,
                                                                          OSMESA_RGBA,
                                                                          OSMESA_DEPTH_BITS,
                                                                          24,
                                                                          OSMESA_STENCIL_BITS,
                                                                          8,
                                                                          OSMESA_PROFILE,
                                                                          OSMESA_COMPAT_PROFILE,
+                                                                         OSMESA_CONTEXT_MAJOR_VERSION,
+                                                                         3,
+                                                                         OSMESA_CONTEXT_MINOR_VERSION,
+                                                                         0,
                                                                          NULL})
                                                  .begin(),
-                                             NULL);
-    }
+                                             NULL)) {}
```

Comment:
```suggestion
                                             nullptr)) {}
```

Might as well update this as well.

Reply:
I'm calling a C function. Should I use `nullptr` instead `NULL` anyway?

Reply:
Yes I don't think there's a good reason to use `NULL` in C++ code, even when calling C libraries.

Reply:
They ultimately mean the same thing, and at least in MSVC/Windows land, NULL is still used in a lot of places. I do agree though a chance to modernize while we're in here is good.